frontend/components/dashboard/chart/SummaryChart.vue

```diff
@@ -155,7 +247,7 @@ const option = computed(() => {
       type: 'value',
       minInterval: 10,
       maxInterval: 20,
-      min: (range: any) => Math.max(0, 10 * Math.ceil(range.min / 10 - 1)),
```

Comment:
Without this lower bound at 0, the scale of the chart shows "-10%" when a validator has been offline, for example:
![image](https://github.com/user-attachments/assets/60042c18-9dd6-4176-bfeb-420363b0dbcb)